sql/data_model.sql

```diff
@@ -0,0 +1,64 @@
+BEGIN;
+
+-- add pgcrypto extension so that we can use the gen_random_uuid function.
+CREATE EXTENSION IF NOT EXISTS pgcrypto;
+
+DROP TABLE IF EXISTS formdata CASCADE;
+CREATE TABLE formdata(
+  id uuid PRIMARY KEY DEFAULT gen_random_uuid(),
+  domain text NOT NULL,
+  received_on timestamp without time zone,
+  time_start timestamp without time zone,
+  time_end timestamp without time zone,
+  duration bigint,
```

Comment:
another minor point and probably out of scope for this PR, but it'd be nice to think hard about what the constraints should be on our data and enforce them at the DB level. like this could have a check to ensure that duration > 0